lab_1/start.py

```diff
@@ -19,6 +19,21 @@
             file_to_read:
         unknown_text = file_to_read.read()
 
+        import main
+
+        en = main.load_profile(os.path.join(PATH_TO_LAB_FOLDER, r"profiles\en.json"))
+        de = main.load_profile(os.path.join(PATH_TO_LAB_FOLDER, r"profiles\de.json"))
+        la = main.load_profile(os.path.join(PATH_TO_LAB_FOLDER, r"profiles\la.json"))
+        new_profile = main.create_language_profile("unk", unknown_text, [])
+        interesting = main.compare_profiles_advanced(new_profile, en, 5)
+        print(interesting)
+        result = main.detect_language_advanced(new_profile, [en, de, la], [], 5)
```

Comment:
can you explain what happens in `start.py`?

Reply:
I was just checking my code c:
The first line creates a new profile for the unknown text
The second line compares profile made from text in unknown language with english profile in this case
The forth line detects what language it is